styles.css

```diff
@@ -112,6 +112,258 @@ body {
   font-weight: 900;
 }
 
+.headline_container ul {
+  list-style: none;
+}
+
+/** portfolio section styles **/
+
+.portfolio_title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 90px 26px;
+}
+
+.portfolio_header {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: #172b4d;
+}
+
+.portfolio_header_underline {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+}
+
+.portfolio_work {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 24px;
+}
+
+.work_item {
+  display: flex;
+  flex-direction: column;
+  margin: 0 26px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.portfolio_image {
+  background: #ebf0ee;
+  height: 252px;
+}
+
+.portfolio_text {
+  background: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  padding: 16px;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.work_title {
+  font-style: normal;
+  font-weight: 600;
+  font-family: "Inter", sans-serif;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+  margin: 0;
+}
+
+.portfolio_techs {
+  display: flex;
+  gap: 8px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.portfolio_techs li {
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+.portfolio_button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+  margin: 14px 0;
+}
+
+.portfolio_button:hover {
+  background: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  transition: all 300 ease-in-out;
+}
+
+.portfolio_button:active {
+  background: #008552;
+}
+
+/* About section */
+
+#about {
+  margin: 68px 0;
+  background-image: url("/images/aboutmetop_desktop.jpg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+.about_top {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  position: relative;
+  padding: 0 12px;
+  background-image: url("/images/aboutbottom-mobile.jpg");
```

Comment:
- [ ] Good job following the previous code reviewer suggestions @Ezzywealth, But till the background are not visible. This is because you haven't use a relative address when you apply the backgrounds. Please remember to add './' for every relative address. Use `background-image: url(./images/aboutbottom-mobile.jpg);` instead of `background-image: url(/images/aboutbottom-mobile.jpg);` . This will tee the browser to look inside the current directory. 

Here I have attached the differences. 
![hum-u](https://user-images.githubusercontent.com/69077061/205267133-fc030cdf-b88e-46d3-bb99-e03997c42ce6.PNG)
![your-icons](https://user-images.githubusercontent.com/69077061/205267140-eb9bd14b-9bc5-46f1-892d-ca7b2ec8647b.PNG)